include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -485,30 +483,25 @@ struct __parallel_transform_scan_static_single_group_submitter<_Inclusive, _Elem
     }
 };
 
-template <typename _Size, ::std::uint16_t _ElemsPerItem, ::std::uint16_t _WGSize, bool _IsFullGroup,
-          typename _KernelName>
+template <typename _Size, typename _KernelName>
 struct __parallel_copy_if_static_single_group_submitter;
 
-template <typename _Size, ::std::uint16_t _ElemsPerItem, ::std::uint16_t _WGSize, bool _IsFullGroup,
-          typename... _ScanKernelName>
-struct __parallel_copy_if_static_single_group_submitter<_Size, _ElemsPerItem, _WGSize, _IsFullGroup,
-                                                        __internal::__optional_kernel_name<_ScanKernelName...>>
+template <typename _Size, typename... _ScanKernelName>
+struct __parallel_copy_if_static_single_group_submitter<_Size, __internal::__optional_kernel_name<_ScanKernelName...>>
```

Comment:
Should we rename this struct to `__parallel_copy_if_dynamic_single_group_submitter` or just `__parallel_copy_if_single_group_submitter`? My understanding from looking at the single work-group scan is that the static is referring to the templated `_ElemsPerItem` and `_WGSize` which is now removed.